code/modules/atmospherics/machinery/components/unary_devices/thermomachine.dm

```diff
@@ -94,6 +102,9 @@
 		calculated_laser_rating += laser.tier
 	min_temperature = max(T0C - (base_cooling + calculated_laser_rating * 15), TCMB) //73.15K with T1 stock parts
 	max_temperature = T20C + (base_heating * calculated_laser_rating) //573.15K with T1 stock parts
+	if(pyro_anomaly_core)
+		max_temperature += 1500
```

Comment:
This should use some temperature define like ``T20C`` above it

Reply:
When I tried doing anything other than T20C I got errors calling them undefined

Reply:
Defined as T1500K in atmos_core.dm due to the fact that it in Celsius would be T1226.85C which is not desirable.